### ISSUE TITLE
Setting useSafeArea to false has no effect

### ISSUE THREAD

**Issue by JamesRandall** [fsprojects/Fabulous#499]
As per the title setting this property has no effect. I've got this Fabulous version of the boilerplate Xaml:

```
let view (model: Application.Model) dispatch =
        View.ContentPage(useSafeArea=false,
                         content=View.FlexLayout(
                                                    direction=FlexDirection.Row,
                                                    justifyContent=FlexJustify.End,
                                                    children=[
                                                        View.Label(text="Welcome to Xamarin.Forms!", horizontalOptions=LayoutOptions.Center, verticalOptions=LayoutOptions.CenterAndExpand)
                                                    ]
                                                ))
```

XAML equivalent:

```
<?xml version="1.0" encoding="utf-8"?>
<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
             xmlns:d="http://xamarin.com/schemas/2014/forms/design"
             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" mc:Ignorable="d" x:Class="safeareatest.MainPage"
             xmlns:ios="clr-namespace:Xamarin.Forms.PlatformConfiguration.iOSSpecific;assembly=Xamarin.Forms.Core"
             ios:Page.UseSafeArea="false">
    <FlexLayout Direction="Row" JustifyContent="End">
        <!-- Place new controls here -->
        <Label Text="Welcome to Xamarin.Forms!" HorizontalOptions="Center" VerticalOptions="CenterAndExpand" />
    </FlexLayout>
</ContentPage>

```
The latter places the label under the status bar while the former (F#) places it under the status bar.

I'm still finding my way around both Fabulous and F# but it looks to me like an issue in how the property is set here:

https://github.com/fsprojects/Fabulous/blob/e1613ad8a1cd7917f871e945fd21049ab8da87f8/Fabulous.XamarinForms/src/Fabulous.XamarinForms.Controls/Xamarin.Forms.Core.json#L1631

If I'm reading that write it's always setting the property to true regardless of what is set in the view code. Fabulous seems to default to the safe area being used and so you can never turn it off.

(I've got Fabulous building so if I get a chance I'll submit a PR - just a bit new to all this)

**Comment by SergejDK** [comment]
Yes you are right. It is always true and does it does not care about setting it.

I don't know the specific reason why it was done this way but I think we should do it as XF does it.
The defaultbehavior is that it is `false` and when needed the dev has to set it to `true`. Normally Fabulous is just a wrapper and we should not change the defaultbehavior.

@TimLariviere do you know the reason why we moved away from XF here?

@JamesRandall 
We always appreciate help on Fabulous! If you have any questions just ask.

**Comment by TimLariviere** [comment]
I just think it was a mistake when it was written. No specific reason for that behavior.

@JamesRandall If you feel up to it, we will gladly merge your PR. Thanks! :)

**Comment by JamesRandall** [comment]
Thanks both! I've just made what I hope is a fix in the style of the project. Tested it with the AllControls sample and it seems to work fine. Took me a little while as I had 30 minutes of "what is going on here... why does the build output look unfamiliar to me" - then I learned about the 0.40 changes :)

I'm fairly new to F# so if there are improvements that could be made to what I've done I'd love to hear them.

Will submit the PR shortly.

Great project by the way. Thank you. I'm fairly new to both F# and Fabulous it (coming from C# on the backend and React / React Native on the front end) but F# has grown on me fast and Fabulous is ticking all my boxes, its a pleasure to work with and a great combo :)
